tests/testthat/test-ensembleMethods.R

```diff
@@ -215,16 +222,20 @@ test_that("Residuals provided by residuals are proper for ensemble objects", {
 
   mr_reg_wide <- as.data.frame(lapply(ens.reg$models, residuals))
   names(mr_reg_wide) <- lapply(ens.reg$models, function(x) x$method)
-  mr_reg_long <- reshape(mr_reg_wide, direction = "long", varying = names(mr_reg_wide),
-                         v.names = "resid", timevar = "method", times = names(mr_reg_wide))
+  mr_reg_long <- reshape(mr_reg_wide,
+    direction = "long", varying = names(mr_reg_wide),
+    v.names = "resid", timevar = "method", times = names(mr_reg_wide)
+  )
   expect_equal(mr_reg_long[order(mr_reg_long$method, mr_reg_long$id), "resid"], mr2[order(mr2$method, mr2$id), "resid"])
 
   ens.class2 <- ens.class
   ens.reg2 <- ens.reg
   ens.class2$modelType <- ens.reg2$modelType <- NULL
 
-  expect_equal(residuals(ens.class2), residuals(ens.class))
-  expect_equal(residuals(ens.reg2), residuals(ens.reg))
+  suppressWarnings({
```

Comment:
What warnings does this supress?

Reply:
`residuals` does not produce any warning. I suppose I made a mistake and thought it produces a warning like `multiResiduals` does.

Reply:
At some point I'll fix that warning. thx https://github.com/zachmayer/caretEnsemble/issues/262